core/src/main/java/org/elasticsearch/transport/TcpTransport.java

```diff
@@ -919,6 +919,7 @@ public void sendRequest(final DiscoveryNode node, final long requestId, final St
                     transportServiceAdapter.onRequestSent(node, requestId, action, request, finalOptions);
                 }
             };
+            Channel targetChannel = nodeChannel(node, options);
```

Comment:
why did you move this? again, original idea is to have the serialization saved when we are disconnected from a node, which most code doesn't check for in advance.